### PR TITLE
trapezoid 4 corner parameters 

### PR DESCRIPTION
**Pull request by SputnikElf**
add 4 corner parameter in html, with horizontal and vertical percent values. new calculation of classic led lines. parameters in advance part. default values are for a rectangle.

### CODE REVIEW

**Comment by Paulchen-Panther** [comment]
Vielen Dank für deinen PR. Ich werde es sicher diese Woche nicht mehr schaffen es zu testen. Über die Feiertage komme ich bestimmt dazu.

File: CrossCompileHowto.txt [hunk 1]
```diff
@@ -63,7 +63,7 @@ echo $HOME/crosscompile/Qt5
 ##### End of Qt5 specific ######
 
 # get the Hyperion sources
-git clone --recursive https://github.com/hyperion-project/hyperion.ng.git "$HYPERION_DIR"
+git clone --recursive https://github.com/SputnikElf/hyperion.ng.git "$HYPERION_DIR"
```
Comment (b1rdhous3) [inline CrossCompileHowto.txt]: Could you please change this back?

Comment (Lord-Grey) [reply CrossCompileHowto.txt]: @b1rdhous3: not sure, if it is possible to pick commits selectively... In my PR #654 I replace the cross-compile.txt file anyway by a pretty-print .md version.

**Comment by moe7863** [comment]
> 4 extended corner points for trapezoid with horizontal and vertical percent value.

I have tested the new feature, 4 corner parameter in html, with horizontal and vertical percent values. Working perfectly, thank you

**Comment by b1rdhous3** [comment]
Why is this not merged? 
Can we solve the conflicts and merge?

**Comment by moe7863** [comment]
This should have been merged ages ago. This has been the most convenient feature for me

**Comment by Paulchen-Panther** [comment]
@SputnikElf 
Could you please resolve the conflicts to bring them together, or will I close your PR and you create a new PR?

**Comment by tpmodding** [comment]
Due to inactivity, we will close this.

**Comment by desertblade** [comment]
If anyone is following this I opened up a new PR #791

**Pull request closed without merging**